src/components/molecules/modals/FollowModal/index.jsx

```diff
@@ -0,0 +1,238 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { Link } from "react-router-dom";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBModal,
+  MDBIcon,
+  MDBModalBody,
+  MDBBtn,
+  MDBModalHeader,
+  MDBRow,
+  MDBCol,
+} from "mdbreact";
+
+//> Stylesheet
+import "./follow.scss";
+//#endregion
+
+//#region > Components
+/** @class A modal that shows a list of follows */
+class FollowModal extends React.Component {
+  state = {
+    loggedUser: null,
+  };
+
+  componentDidMount = () => {
+    const { loggedUser } = this.props;
+
+    this.setState({
+      loggedUser: loggedUser,
+    });
+  };
+
+  componentDidUpdate = () => {
+    const { loggedUser } = this.props;
+
+    if (loggedUser !== this.state.loggedUser) {
+      this.setState({
+        loggedUser: loggedUser,
+      });
+    }
+  };
+
+  follow = (personToFollow) => {
+    if (!this.props.loggedUser?.anonymous) {
+      this.props.follow(personToFollow).then(() => {
+        let loggedUser = this.props.loggedUser;
+        let follows = [];
+
+        for (let count in loggedUser.person.follows) {
+          follows.push(loggedUser.person.follows[count]);
+        }
+
+        follows.push({ slug: "p-" + personToFollow });
+
+        loggedUser.person.follows = follows;
+
+        this.setState({ loggedUser });
+      });
+    } else {
+      this.props.toContinue();
+    }
+  };
+
+  unfollow = (personToUnfollow) => {
+    this.props.unfollow(personToUnfollow).then(() => {
+      let loggedUser = this.state.loggedUser;
+      let follows = [];
+
+      for (let count in loggedUser?.person?.follows) {
+        if (loggedUser.person.follows[count].slug !== "p-" + personToUnfollow) {
+          follows.push(loggedUser.person.follows[count]);
+        }
+      }
+
+      loggedUser.person.follows = follows;
```

Comment:
This should be reworked:

- Use the new state defined above
- Rework the remove algorithm as follows:
```
   var members = [
        {name: 'Anna', class: 'one'}, 
        {name: 'Bob', class: 'two'},  
        {name: 'Chuck', class: 'two'}];

   var idx = members.findIndex(p => p.class=="two");
   var removed = members.splice(idx,1);
     
   console.log(removed);
   console.log(members);
```